vllm/model_executor/models/minicpmv.py

```diff
@@ -387,7 +451,7 @@ def get_embedding(
             if len(image_bounds) > 0:
                 image_indices = torch.stack([
                     torch.arange(start, end, dtype=torch.long)
-                    for start, end in image_bounds.tolist()
+                    for start, end in image_bounds.squeeze(0).tolist()
```

Comment:
To keep things consistent, let's flatten `image_bounds` inside `_parse_and_validate_input`.